src/mds/CInode.h

```diff
@@ -60,6 +61,52 @@ struct cinode_lock_info_t {
   int wr_caps;
 };
 
+struct CInodeCommitOperation : public ObjectOperation {
```

Comment:
move this into .cc file

Reply:
The Journal.cc will need the `CInodeCommitOperation` and if it is not full declared in the header file, the build will fail.

Reply:
no need to inherit from ObjectOperation. just define a ObjectOperation in CInodeCommitOperation::update

Reply:
Yeah, will try to fix it.